src/frequenz/sdk/microgrid/grid.py

```diff
@@ -0,0 +1,80 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Grid connection point.
+
+This module provides the `Grid` type, which represents a grid connection point
+in a microgrid.
+"""
+
+import logging
+from dataclasses import dataclass
+from typing import Iterable, Optional
```

Comment:
New import location for `Iterable` (and `Optional` is not necessary)

```suggestion
from collections.abc import Iterable
```

Reply:
Done

Reply:
isn't typing the new location?

Reply:
Lemme see what pylance suggests.

Reply:
pylance suggests `typing`. I changed it to `typing`.

Reply:
Sorry, but `pylance` is wrong :)

Python provided the stuff in `collections.abc` but at first it could just use regular types as type hints, so they needed to add special types to the `typing` module (there was even a `typing.Int`), but then it got the ability to use regular types, the use of types in `typing` become deprecated.

Reply:
reverted

Reply:
Thanks! Just for more context:

![image](https://github.com/frequenz-floss/frequenz-sdk-python/assets/1031485/b9f99c93-0f37-4ded-bec7-d62be0be2389)